src/main/java/com/google/sps/data/Places.java

```diff
@@ -198,22 +198,7 @@
             PlaceDetailsRequest.FieldMask.WEBSITE)
         .await();
 
-        hiddenGems.add(new HiddenGem(
-          place.placeId, 
-          place.name, 
-          place.types,  
-          placeDetails.formattedAddress, 
-          place.geometry.location.lat, 
-          place.geometry.location.lng, 
-          String.valueOf(placeDetails.priceLevel), 
-          place.rating, 
-          place.userRatingsTotal, 
-          placeDetails.website, 
-          place.openingHours, 
-          place.photos[0].photoReference, 
-          place.photos[0].htmlAttributions, 
-          place.permanentlyClosed, 
-          place.businessStatus));
+        hiddenGems.add(new HiddenGem(place, placeDetails));
```

Comment:
this might be a silly question but why did you remove all the place details?

Reply:
not a silly question at all. I haven't removed the place details, I send the enter place details object as a param and then set the relevant values in the hidden gem object. like hidden.priceLevel = placeDetails.priceLevel.

Reply:
makes sense ! thanks for explaining